laser_encoders/download_models.py

```diff
@@ -0,0 +1,155 @@
+                for chunk in response.iter_content(chunk_size=1024):
+                    f.write(chunk)
+                    progress_bar.update(len(chunk))
+            progress_bar.close()
+
+    def download_laser2(self):
+        self.download(f"{self.url}/laser2.pt")
+        self.download(f"{self.url}/laser2.spm")
+        self.download(f"{self.url}/laser2.cvocab")
+
+    def download_laser3(self, lang: str, version: str = "v1", spm: bool = False):
+        lang_3_4 = LASER3_LANGUAGE[lang]
+        if isinstance(lang_3_4, tuple):
+            options = ", ".join(f"'{opt}'" for opt in lang_3_4)
+            logger.info(
+                f"Language '{lang_3_4}' has multiple options: {options}. Please specify using --lang-model."
+            )
+            return
+
+        self.download(f"{self.url}/laser3-{lang_3_4}.{version}.pt")
+        if spm:
+            if lang_3_4 in SPM_LANGUAGE:
+                self.download(f"{self.url}/laser3-{lang_3_4}.{version}.spm")
+                self.download(f"{self.url}/laser3-{lang_3_4}.{version}.cvocab")
+            else:
+                self.download(f"{self.url}/laser2.spm")
+                self.download(f"{self.url}/laser2.cvocab")
+
+    def main(self, args):
+        if args.lang_model in LASER2_LANGUAGE:
+            self.download_laser2()
+        elif args.lang_model in LASER3_LANGUAGE:
+            self.download_laser3(
+                lang=args.lang_model, version=args.version, spm=args.spm
+            )
+        else:
+            raise ValueError(
+                f"Unsupported language name: {args.lang_model}. Please specify a supported language name using --lang-model."
+            )
+
+
+def download_model(model_dir, version, lang_model, spm):
+    if model_dir is None:
+        model_dir = os.path.expanduser("~/.cache/laser_encoders")
+    file_path = ""
+    downloader = LaserModelDownloader(model_dir)
+    if lang_model in LASER2_LANGUAGE:
+        downloader.download_laser2()
+        file_path = model_dir + "/laser2"
+    elif lang_model in LASER3_LANGUAGE:
+        downloader.download_laser3(lang=lang_model, version=version, spm=spm)
+        file_path = f"{model_dir}/laser3-{lang_model}.{version}"
+    else:
+        raise ValueError(
+            f"Unsupported language name: {lang_model}. Please specify a supported language name using --lang-model."
+        )
+    return file_path
+
+
+def initialize_encoder(lang_model, model_dir=None, version="v1", spm=False):
+    file_path = download_model(model_dir, version, lang_model, spm)
+    model_path = f"{file_path}.pt"
+    spm_path = f"{file_path}.cvocab"
+    if not os.path.exists(spm_path):
+        model_dir, _ = os.path.split(spm_path)
+        spm_path = f"{model_dir}/laser2.cvocab"
+    return SentenceEncoder(model_path=model_path, spm_vocab=spm_path)
+
+
+def initialize_tokenizer(lang_model, model_dir=None, version="v1", spm=False):
+    file_path = download_model(model_dir, version, lang_model, spm)
```

Comment:
It seems that we download a whole large `.pt` model here, but don't use it when initializing the tokenizer. 
Can you download only the spm part here?